web/client/utils/StateUtils.js

```diff
@@ -125,7 +125,7 @@ const fetchEpic = (storeName = PERSISTED_STORE_NAME, name = 'rootEpic') => {
  * @param {string} name optional name (if you want to persist more than one store)
  */
 export const getState = (name) => {
-    return getStore(name) && getStore(name).getState() || {};
+    return getStore(name)?.getState?.() ?? {};
```

Comment:
i think other falsy values are useful here, don't they?
for example if getState returns for a strange reason "" or 0, would be better to return a {} right instead of them.

i know i know, probably i'm doing too much mental saws

Reply:
The purpose of ?? is to avoid null and undefined. 
Store getState returns an object, not a number or a string, so I think ?? is the right choice here.